src/domain/Game.java

```diff
@@ -0,0 +1,93 @@
+package domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Game {
+    private List<Line> ladder;
+    private List<String> players;
+    private List<String> results;
+    private GamePlayResults gamePlayResults;
+    private int nowPlayerIndex;
+    private int result;
+
+    public Game(Ladder ladder){
```

Comment:
항상 포매팅되도록 해주세요.

Reply:
여기서 포매팅하라는 말의 뜻이 잘 이해가 가지않습니다. 설명을 부탁드려도 될까요?

Reply:
코드 포매팅입니다.
```suggestion
    public Game(Ladder ladder) {
```
이와 같이 손으로 쓰다보면 실수가 생기는데 리포맷 단축키를 종종 눌러서 포맷을 맞춰주세요~